### PR TITLE
refactor: design auth view and add intro view

### PR DESCRIPTION
**Pull request by urielPalomares**
Captura de pantalla 2023-03-09 a la(s) 13 44 29
Captura de pantalla 2023-03-09 a la(s) 13 44 40
Captura de pantalla 2023-03-09 a la(s) 13 44 53
Captura de pantalla 2023-03-09 a la(s) 13 45 05

### CODE REVIEW

File: components/auth/auth.test.tsx [hunk 1]
```diff
@@ -10,14 +10,14 @@ describe('Auth', () => {
...
-    expect(spotifyLogo).toBeInTheDocument();
+    expect(connectionLogo).toBeInTheDocument();
   });
 
   it('Should render call to action buttons', () => {    
-    const loginButton = screen.getByText('Login');
+    const loginButton = screen.getByText('Sign up');
     expect(loginButton).toBeInTheDocument();
   });
 })
```
Comment (Andres2D) [inline components/auth/auth.test.tsx:23]: Add emppty line at the end of the file

File: components/auth/auth.tsx [hunk 2]
```diff
@@ -1,27 +1,46 @@
...
-    <div className={styles.card}>
-      <Image 
+    <div
+      className={styles.home}
+      style={{
+        backgroundColor: 'teal',
+        width: '100%',
+        height: '100vh',
+        objectFit:'cover',
+      }}
```
Comment (Andres2D) [inline components/auth/auth.tsx:20]: Remove this from here, and add it to the scss file.

File: components/auth/auth.tsx [hunk 3]
```diff
@@ -1,27 +1,46 @@
...
-        colorScheme='blackAlpha'
-        onClick={() => signIn('spotify_user')}>
-          Login
-      </Button>
+
+      <ButtonGroup>
+        <Button
+        style={{
+          backgroundColor: '#123837'
+        }}
```
Comment (Andres2D) [inline components/auth/auth.tsx:37]: Same here.

File: components/intro/intro.module.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  align-items: center;
+}
+
+.logo {
+  margin-bottom: 30px;
+  width: 250px;
+}
+
+.version {
+  color: #123837;
```
Comment (Andres2D) [inline components/intro/intro.module.scss:16]: Add this to the colors scss variables file.

File: components/intro/intro.test.tsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+  beforeEach(() => {
+    render(<IntroLayout />);
+  })
+
+  it('Should render logo playlist ,aster', () => {
+    const appLogo = screen.getByAltText('playlist-master'); 
+    expect(appLogo).toBeInTheDocument();
+  });
+})
```
Comment (Andres2D) [inline components/intro/intro.test.tsx:20]: Add empty line at the end of the file.

File: components/intro/intro.tsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  const router = useRouter();
+
+  return (
+    <div
+    className={styles.home}
+      style={{
+        backgroundColor: 'teal',
+        width: '100%',
+        height: '100vh',
+      }}
```
Comment (Andres2D) [inline components/intro/intro.tsx:17]: Use this on the scss file.

File: pages/index.tsx [hunk 7]
```diff
@@ -11,7 +11,7 @@ export const getServerSideProps = async(context: any) => {
 
   return {
     redirect: {
-      destination: session ? '/menu' : '/auth',
+      destination: session ? '/menu' : '/intro',
```
Comment (Andres2D) [inline pages/index.tsx:14]: How we'll we go from intro to auth?

File: pages/intro/index.tsx [hunk 8]
```diff
@@ -0,0 +1,11 @@
+import { NextPage } from 'next';
+import IntroLayout from '../../components/intro/intro';
+
+const intro: NextPage = () => {
```
Comment (Andres2D) [inline pages/intro/index.tsx:4]:
```suggestion
const Intro: NextPage = () => {
```

**Review by Andres2D — CHANGES_REQUESTED** [review]

File: components/auth/auth.tsx (hunk 2 above)
Comment (urielPalomares) [reply components/auth/auth.tsx:20]: Si los paso al file de scss no toma los cambios bro,

File: components/auth/auth.tsx (hunk 3 above)
Comment (urielPalomares) [reply components/auth/auth.tsx:37]: igualemente, si los paso al scss no toma los cambios

File: components/intro/intro.module.scss (hunk 4 above)
Comment (urielPalomares) [reply components/intro/intro.module.scss:16]: Listo

File: pages/index.tsx (hunk 7 above)
Comment (urielPalomares) [reply pages/index.tsx:14]: con el evento click que se le declaro en el intro.tsx

File: components/auth/auth.test.tsx (hunk 1 above)
Comment (urielPalomares) [reply components/auth/auth.test.tsx:23]: Ready

File: components/intro/intro.tsx (hunk 6 above)
Comment (urielPalomares) [reply components/intro/intro.tsx:17]: Igual

File: pages/intro/index.tsx (hunk 8 above)
Comment (urielPalomares) [reply pages/intro/index.tsx:4]: Ready

File: components/intro/intro.module.scss [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+  width: 100%;
+  height: 100vh;
+}
+
+.logo {
+  margin-bottom: 30px;
+  width: 250px;
+}
+
+.version {
```
Comment (Andres2D) [inline components/intro/intro.module.scss:18]: Use the sass nesting

````
.home {
 ...
  .logo {
    ...
   }
}
```

**Review by Andres2D — APPROVED** [review]

File: components/auth/auth.tsx (hunk 2 above)
Comment (urielPalomares) [reply components/auth/auth.tsx]: Listo

File: components/intro/intro.tsx (hunk 6 above)
Comment (urielPalomares) [reply components/intro/intro.tsx]: Listo

File: components/intro/intro.module.scss (hunk 9 above)
Comment (urielPalomares) [reply components/intro/intro.module.scss]: Ready

File: components/auth/auth.tsx (hunk 3 above)
Comment (urielPalomares) [reply components/auth/auth.tsx]: Ready

**Review by Andres2D — APPROVED** [review]

**Pull request merged**